enterprise/frontend/src/metabase-enterprise/settings/types.ts

```diff
@@ -0,0 +1,23 @@
+import { Settings } from "metabase-types/api";
+import { State } from "metabase-types/store";
+
+export interface EnterpriseState extends State {
+  settings: EnterpriseSettingsState;
+}
+
+interface EnterpriseSettingsState {
+  values: EnterpriseSettings;
+}
+
+export interface EnterpriseSettings extends Settings {
+  "application-colors"?: Record<string, string>;
+  "application-logo-url"?: string;
+  /**
+   * @deprecated
+   */
+  application_logo_url?: string;
```

Comment:
I believed the settings name in snake_case is outdated. Correct me if I'm wrong.